lamp_kotlin/build.gradle

```diff
@@ -49,4 +49,16 @@ dependencies {
     //activity recognition.
     implementation 'com.google.android.gms:play-services-location:17.1.0'
 
+}
+
+android {
```

Comment:
Please remove this. We do not want to publish to Maven. Use jitpack.io and mark the dependency as `compile` instead of `implementation`.